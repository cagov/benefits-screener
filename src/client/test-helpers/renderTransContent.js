import React from "react";
import { mount } from "enzyme";
import { I18nextProvider } from "react-i18next";
import i18n from "./i18nForTests";

function renderTransContent(Component, props = {}) {
  const wrapper = mount(
    <I18nextProvider i18n={i18n}>
      <Component {...props} />
    </I18nextProvider>
  );

  // Filter out the very verbose I18nextProvider component wrapper
  return wrapper.find(Component);
}

module.exports = renderTransContent;
