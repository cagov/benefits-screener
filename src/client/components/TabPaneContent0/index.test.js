import renderTransContent from "../../test-helpers/renderTransContent";
import TabPaneContent from "./index";

describe("<TabPaneContent0 />", () => {
  it("renders the TabPaneContent component", async () => {
    const props = {
      getTabLink: jest.fn(),
    };
    const content = renderTransContent(TabPaneContent, props);
    expect(content).toMatchSnapshot();
  });
});
