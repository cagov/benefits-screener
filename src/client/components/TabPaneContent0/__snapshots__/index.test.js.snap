// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPaneContent0 /> renders the TabPaneContent component 1`] = `
<TabPaneContent0
  getTabLink={
    [MockFunction] {
      "calls": Array [
        Array [
          4,
        ],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
>
  <div>
    <div
      className="gray-box mt-0"
    >
      <h3>
        <Trans
          i18nKey="tab0-header1"
          t={[Function]}
        >
          Tips for applying on UI Online
        </Trans>
      </h3>
      <ul>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item1"
            t={[Function]}
          >
            Everyone (UI and PUA applicants) uses the same form. 
            <a
              href="https://www.edd.ca.gov/Benefit_Programs_Online.htm"
              key="1"
            >
              Register or log in with Benefit Programs Online
            </a>
             and click “File New Claim.”
          </Trans>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item2"
            t={[Function]}
          >
            File a new claim if you have already applied for UI but are eligible for PUA because you are a business owner, independent contractor, self-employed worker, freelancer, or gig worker affected by COVID-19, 
            <strong
              key="strong-1"
            >
              and
            </strong>
             you have 
            <strong
              key="strong-3"
            >
              not
            </strong>
             already appealed a $0 award notice, been asked to verify your identity, or requested a wage investigation.
          </Trans>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item3"
            t={[Function]}
          >
            If you are a small business owner, independent contractor, self-employed worker, freelancer, or gig worker, here’s how to answer key questions in the application form:
          </Trans>
          <ul>
            <li>
              <Trans
                i18nKey="tab0-list1a-item1"
                t={[Function]}
              >
                On the Employment History screen when you supply your last employer information, select "No."
              </Trans>
            </li>
            <li>
              <Trans
                i18nKey="tab0-list1a-item2"
                t={[Function]}
              >
                On the Availability Information page, answer question 7 with "No."
              </Trans>
            </li>
            <li>
              <Trans
                i18nKey="tab0-list1a-item3"
                t={[Function]}
              >
                On the Disaster Information page, answer question 1a.3 with “You are an independent contractor.” If you got paid in cash, select “None of these options apply to me.”
              </Trans>
            </li>
          </ul>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item4"
            t={[Function]}
          >
            If you have run out of benefits, you are unemployed, and your benefit year has expired, reapply for benefits through UI Online.
          </Trans>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item5"
            t={[Function]}
          >
            If you have run out of benefits but you are still within the one year that your claim is good for, and your benefits:
          </Trans>
          <ul>
            <li>
              <Trans
                i18nKey="tab0-list1b-item1"
                t={[Function]}
              >
                Ran out 
                <strong
                  key="strong-1"
                >
                  before February 2
                </strong>
                , you are probably not eligible for PUA. (You would have to show that you are unemployed because of COVID-19.) You may be eligible for the federal 13-week extension. We will notify you about what to do.
              </Trans>
            </li>
            <li>
              <Trans
                i18nKey="tab0-list1b-item2"
                t={[Function]}
              >
                Were for weeks you spent unemployed 
                <strong
                  key="strong-1"
                >
                  between February 2 and March 21
                </strong>
                , we will send you a form to fill out to supply information not already in your claim. When you get the form, fill it out and return it.
              </Trans>
            </li>
          </ul>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item6"
            t={[Function]}
          >
            If you received your last payment for weeks you were unemployed 
            <strong
              key="strong-1"
            >
              on or after March 21
            </strong>
             and are still within your benefit year, we will automatically extend your claim for 13 weeks. We will notify you that we have done that and when you can certify.
          </Trans>
        </li>
        <li
          className="mb-3"
        >
          <Trans
            i18nKey="tab0-list1-item7"
            t={[Function]}
          >
            If you applied for unemployment assistance but didn’t qualify (for example, because you voluntarily quit or you were fired or discharged by your employer), or you are serving a penalty because of false statements on a past claim, you could be eligible for PUA. We will send you a form to fill out to supply information not already in your account. When you get the form, fill it out and return it.
          </Trans>
        </li>
      </ul>
    </div>
    <h3>
      <Trans
        i18nKey="tab0-header2"
        t={[Function]}
      >
        Benefits you can apply for
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab0-p1"
        t={[Function]}
      >
        These benefits are available for California residents. If you are still unsure of your eligibility or what to apply for, 
        <strong
          key="strong-1"
        >
          apply anyway
        </strong>
         as soon as you are out of work or had your hours reduced. Complete the application as best you can, and we will follow up with you as soon as possible.
      </Trans>
    </p>
    <h3>
      <Trans
        i18nKey="tab0-header3"
        t={[Function]}
      >
        Unemployment Insurance (UI)
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab0-p2"
        t={[Function]}
      >
        You may be eligible for regular UI if you have income reported on a W2 and you meet 
        <strong
          key="strong-1"
        >
          any
        </strong>
         of these:
      </Trans>
    </p>
    <ul>
      <li>
        <Trans
          i18nKey="tab0-list2-item1"
          t={[Function]}
        >
          Your employment status has been affected by COVID-19 and you are fully or partially unemployed as a result.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list2-item2"
          t={[Function]}
        >
          You are fully or partially unemployed. This includes layoffs, furloughs, reduced wages, or reduced hours. You can still receive unemployment benefits while working, depending on your pay.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list2-item3"
          t={[Function]}
        >
          Your child’s school is closed, and you need to miss work to care for them.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list2-item4"
          t={[Function]}
        >
          Your previous UI claim has expired.
        </Trans>
      </li>
    </ul>
    <p>
      <Trans
        i18nKey="tab0-p3"
        t={[Function]}
      >
        If you're already receiving UI, review 
        Receive your benefits
         to learn how your UI claim may be affected by COVID-19.
      </Trans>
    </p>
    <h3>
      <Trans
        i18nKey="tab0-header4"
        t={[Function]}
      >
        Pandemic Unemployment Assistance (PUA)
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab0-p4"
        t={[Function]}
      >
        If you are a business owner, independent contractor, self-employed worker, freelancer, or gig worker and only received a 1099 tax form last year, you are most likely eligible for PUA.
      </Trans>
    </p>
    <h4>
      <Trans
        i18nKey="tab0-header5"
        t={[Function]}
      >
        Apply for PUA if your work situation changed because of COVID-19, and you meet 
        <strong
          key="strong-1"
        >
          any
        </strong>
         of these:
      </Trans>
    </h4>
    <ul>
      <li>
        <Trans
          i18nKey="tab0-list3-item1"
          t={[Function]}
        >
          You had a definite date to begin work, but the job is no longer available, or you could not reach the job as a direct result of COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list3-item2"
          t={[Function]}
        >
          You are unable to travel to your job as a direct result of COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list3-item3"
          t={[Function]}
        >
          You quit your job as a direct result of COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list3-item4"
          t={[Function]}
        >
          Your workplace is closed as a direct result of COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list3-item5"
          t={[Function]}
        >
          You are unemployed, partially employed, or unable to work because COVID-19 has forced you to stop working.
        </Trans>
      </li>
    </ul>
    <h4>
      <Trans
        i18nKey="tab0-header6"
        t={[Function]}
      >
        Apply for PUA if you have health issues because of COVID-19, and you meet any of these:
      </Trans>
    </h4>
    <ul>
      <li>
        <Trans
          i18nKey="tab0-list4-item1"
          t={[Function]}
        >
          You have been diagnosed with COVID-19 or are experiencing symptoms and are seeking a medical diagnosis.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list4-item2"
          t={[Function]}
        >
          A member of your household has been diagnosed with COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list4-item3"
          t={[Function]}
        >
          You are caring for a family member or a member of your household who has been diagnosed with COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list4-item4"
          t={[Function]}
        >
          You are caring for a dependent during the work day because their school or another care facility has closed due to COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list4-item5"
          t={[Function]}
        >
          Your healthcare provider has told you to self-quarantine because of COVID-19.
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list4-item6"
          t={[Function]}
        >
          You have become the main income provider due to a COVID-19 death in your household.
        </Trans>
      </li>
    </ul>
    <h3>
      <Trans
        i18nKey="tab0-header7"
        t={[Function]}
      >
        Disability Insurance (DI) or Paid Family Leave (PFL)
      </Trans>
    </h3>
    <p>
      <Trans
        i18nKey="tab0-p5"
        t={[Function]}
      >
        If you’re not eligible for UI, you may be eligible for Disability Insurance or Paid Family Leave. You usually qualify if you paid into the State Disability Insurance program (noted as “CASDI” on paystubs), via taxes.
      </Trans>
    </p>
    <ul>
      <li>
        <Trans
          i18nKey="tab0-list5-item1"
          t={[Function]}
        >
          If you are unable to perform your normal work duties because you are sick or quarantined, apply for 
          <a
            href="https://edd.ca.gov/Disability/How_to_File_a_DI_Claim_in_SDI_Online.htm"
            key="1"
          >
            Disability Insurance.
          </a>
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="tab0-list5-item2"
          t={[Function]}
        >
          If you are caring for a family member who is sick, apply for 
          <a
            href="https://edd.ca.gov/Disability/How_to_File_a_PFL_Claim_in_SDI_Online.htm"
            key="1"
          >
            Paid Family Leave.
          </a>
        </Trans>
      </li>
    </ul>
  </div>
</TabPaneContent0>
`;
